style_suppressions.xml

```diff
@@ -5,9 +5,8 @@
         "https://checkstyle.org/dtds/suppressions_1_2.dtd">
 
 <suppressions>
-    <!-- Don't do any CheckStyle checks on any Test files. -->
-    <suppress checks="[a-zA-Z0-9]*"
-              files="com[\\/]easypost[\\/].*Test.java"/>
+    <!-- Don't enforce MagicNumber checks on unit test files -->
+    <suppress checks="MagicNumber" files="com[\\/]easypost[\\/].*Test.java"/>
```

Comment:
e.g. CheckStyle complains about hard-coded numbers like `Thread.sleep(10000)`

Reply:
You could just assign the 10000 to a variable heh. Magic numbers are just loose ints that aren't descriptive. I'd be in favor of keeping the check and doing the right thing by the linter

Reply:
Well, it also complains about other magic numbers like in some of the assertions.

Reply:
All the magic number complaints:
<img width="443" alt="image" src="https://user-images.githubusercontent.com/17054780/171913274-0fe19164-aa2f-4980-b05d-881fbccc9bc2.png">

Reply:
Woof. Ok, I’m fine if test files ignore magic numbers, but the actual library files should definitely not.